integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/allqueries/AllDataTypesTestCaseAggregate.scala

```diff
@@ -59,21 +59,4 @@ class AllDataTypesTestCaseAggregate extends QueryTest with BeforeAndAfterAll {
       Seq(Row(100005.8)))
   })
 
-  test("CARBONDATA-60-union-defect")({
```

Comment:
Because the previous builder 559 added one test case, so the builder 560 has two deleted test case.